manifests/00_namespace.yaml

```diff
@@ -6,3 +6,13 @@ metadata:
     openshift.io/node-selector: ""
   labels:
     openshift.io/cluster-monitoring: "true"
+
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-csi-snapshot-controller
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-monitoring: "true"
```

Comment:
Why did we move this here?

Reply:
I thought this was discussed during one of the meetings this week. From what I recall, the consensus on the meeting was that it'd be better to put the controller resources into `manifests/` so that they're initialized by CVO instead of defining separate functions to manually add them?